src/utils/getUserById.tsx

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. Consider refactoring your code to pass the `users` data as a prop to the components that require it, instead of importing it directly within utility functions.